iris.cabal

```
@@ -122,11 +125,15 @@ test-suite iris-test
   other-modules:
     Test.Iris
       Test.Iris.Tool
+      Test.Iris.Cli
+    Paths_iris
+      
 
   build-depends:
     , iris
     , hspec >= 2.9.7 && < 2.11
     , text
+    , optparse-applicative ^>= 0.17
```

You can actually omit the dependency here. The test suite depends on the library so it'll use the version that satisfies the same constraints.

Removing the constraint in tests will make it easier in the future to bump the dependency: fewer changed lines 🙂 
```suggestion
    , optparse-applicative
```

---

Fixed